src/content/guides/scaffolding.md

```diff
@@ -7,17 +7,21 @@ contributors:
   - EugeneHlushko
 ---
 
-It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance is quite hard. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
+It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance could be even more difficult. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
 
 ## Creating a scaffold
 
-Before writing a `webpack-cli` scaffold, think about what you're trying to achieve. Do you want a "general" scaffold that could be used by any project or type of app? Do you want something focused, like a scaffold that writes both your `webpack.config.js` and your framework code? It's also useful to think about the user experience for your scaffold.
+Before writing a `webpack-cli` scaffold, think about what you're trying to achieve and who is going to use it:
+
+- Do you want a generalized scaffold that could be used by any project or type of application?
```

Comment:
```suggestion
- Do you want a generalized scaffold that could be used in wide variety of applications and projects`
```

Reply:
hmm that makes sense although it should be 
```suggestion
- Do you want a generalized scaffold that could be used by a variety of applications and projects?
```

Reply:
i think `for wide`  should become `a`

Reply:
Quantification is important, thus wide should be there.
```suggestion
- Do you want a generic scaffold that could be used by a wide variety of applications and projects?
```